src/backend/api/bucket.py

```diff
@@ -93,6 +70,9 @@ async def get_bucket_count(conn: S3GWClientDep) -> int:
 async def create_bucket(
     conn: S3GWClientDep, bucket_name: str, enable_object_locking: bool = False
 ) -> None:
+    """
+    See https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3/client/create_bucket.html
```

Comment:
I think #232 will see it like [this](https://github.com/aquarist-labs/s3gw-ui/pull/232/files#diff-df1334458db44af6ba34af611e6af6a858a2d43d4704a8f6f14fb762d94b7ee9R71):
```suggestion
    See
    https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3/client/create_bucket.html
```

Reply:
we can wait and merge this after https://github.com/aquarist-labs/s3gw-ui/pull/232, I will rebase on that.

Reply:
yeah, that would be preferable IMO